src/core/config/config.rs

```diff
@@ -209,6 +210,85 @@ pub struct RootSchema {
 /// Used to omit a field from public consumption.
 pub struct Omit {}
 
+// generate Resolver with macro in order to autogenerate conversion code
+// from the underlying directives.
+// TODO: replace with derive macro
```

Comment:
Create an issue for this.

Reply:
https://github.com/tailcallhq/tailcall/issues/2695